src/main/java/org/traccar/protocol/TotemProtocol.java

```diff
@@ -25,15 +25,20 @@
 public class TotemProtocol extends BaseProtocol {
 
     public TotemProtocol() {
-        setSupportedDataCommands(
+        String[] supportedCommands = new String[]{
```

Comment:
Please just duplicate this in each call. List changes pretty often and most of the time it doesn't match, so makes sense to have a little bit of duplication.